extensions/prettier/package.json

```diff
@@ -0,0 +1,151 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "prettier",
+  "title": "Prettier Markdown",
+  "description": "Format code with Prettier",
+  "icon": "prettier-logo.png",
+  "author": "lin",
+  "categories": [
+    "Developer Tools"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Format Code",
+      "subtitle": "Apply Prettier and Markdown",
+      "description": "Apply Prettier to Clipboard content",
+      "mode": "no-view"
+    }
+  ],
+  "preferences": [
+    {
+      "name": "parser",
+      "type": "dropdown",
+      "title": "Select parser",
+      "description": "Specify which parser to use.",
+      "label": "Include AirPlay devices",
```

Comment:
Airplay? Do we need to send `parser` to Prettier? It should work [automatically](https://prettier.io/docs/en/options.html#parser)

Reply:
you're right, label removed.

Reply:
Thanks - Do we need to send `parser` to Prettier? It should work [automatically](https://prettier.io/docs/en/options.html#parser)

Reply:
"Prettier automatically infers the parser from the input file PATH, so you shouldn’t have to change this setting."

It works automatically when applied to a file. It would infer the parser from the file extension (ex. .js, .html...)
Because it is parsing the code from the clipboard. It does not know which parser to use.